js-oop/game/game.js

```diff
@@ -0,0 +1,21 @@
+function GameObject(imageUrl) {
+    this.imageUrl = imageUrl;
+    this.ready = false;
+    this.x = 0;
+    this.y = 0;
+}
+
+GameObject.prototype.image = new Image();
+GameObject.prototype.onload = function () {
```

Comment:
onload belongs to Image, not to GameObject

Reply:
Thank you for your comments, I tried to fix it but still not sure about onload()